app/webpacker/css/admin/components/buttons.scss

```diff
@@ -1,6 +1,6 @@
 input[type="submit"],
-input[type="button"],
-button:not(.plain),
+input[type="button"]:not(.trix-button),
+button:not(.plain):not(.trix-button),
```

Comment:
I personally don't like the `not` selector. It creates a dependency. The general class has to know about all the exemptions instead of the exemptions defining themselves. Anyway, that's just my opinion. :stuck_out_tongue: 

Just wondering if a growing list of `not` is a code smell. Maybe we shouldn't blindly style all buttons without adding a class to them?